examples/snap-plugin-collector-rand/rand/rand.go

```diff
@@ -79,7 +92,15 @@ func (RandCollector) CollectMetrics(mts []plugin.Metric) ([]plugin.Metric, error
 		if val, err := mt.Config.GetBool("testbool"); err == nil && val {
 			continue
 		}
-		if mt.Namespace[len(mt.Namespace)-1].Value == "integer" {
+
+		if mt.Namespace[0].Value == "static" {
```

Comment:
This section isn't parsing the same data as the other sections. This section is for getting the input (in this example `something`) from the config, where the config looks something like this: `'{"value":"something"}'`